src/components/Forms/User/User.tsx

```diff
@@ -80,7 +80,17 @@ function UserForm(props: UserProps) {
 				uploadResponse = await uploadFile(formData);
 			}
 			if (verifyAndUpdateUserForm) {
-				console.log("QUERRY");
+				updateUser({
+					variables: {
+						id: value.id,
+						input: {
+							name: value.name,
+							password: value.password,
+							profile_photo:
+								value.uploadPhoto === "removed" ? null : uploadResponse?.[0]?.id,
```

Comment:
why are we matching with a string directly?

Reply:
will change it to enum